### ISSUE TITLE
Awkward icon with spinners

### ISSUE THREAD

**Issue by davidascher** [mozilla/advocacy.mozilla.org#11]
The lack of color contrast between the spinner icons and the (seemingly off-brand) red circle is jarring, IMO:

![screenshot 2015-04-07 10 59 56](https://cloud.githubusercontent.com/assets/51278/7029863/493b19ca-dd15-11e4-9cc9-530b4a14ffa5.png)

**Comment by ScottDowne** [comment]
New site removed all this.
